src/build/build_helpers.jl

```diff
@@ -280,4 +269,9 @@ end
 function get_final_demand_name(demand::Demand)
     demand_function = demand.demand_function::DemandFunction
     return Symbol("$(get_name(demand.commodity, true))ρ$(get_demand_func_name(demand_function))")
-end
\ No newline at end of file
+end
+
+function get_comp_demand_name(i::Input)
+    p = i.production_function::Production 
+    return Symbol("$(get_name(i.commodity, true))†$(get_prod_func_name(p))")
+end
```

Comment:
Just a re-order here, to match the order in build_market_clearance.jl (which is primarily where these functions are called)